cassandra/src/test/scala/akka/stream/alpakka/cassandra/scaladsl/CassandraSourceSpec.scala

```diff
@@ -106,7 +107,7 @@ class CassandraSourceSpec extends WordSpec with ScalaFutures with BeforeAndAfter
     "stream the result of Cassandra statement that results in no data" in {
       val stmt = new SimpleStatement("SELECT * FROM akka_stream_scala_test.test")
 
-      val rows = CassandraSource(stmt)
+      val rows = CassandraSourceStage(stmt)
```

Comment:
This and above renames are probably collateral damage. Should stay `CassandraSource`. This is what fails PR validation.